internal/graphengine/engine_v2.go

```diff
@@ -214,6 +216,17 @@ func (e *EngineV2) ProcessAndStoreGraphQLRequest(w http.ResponseWriter, r *http.
 	if e.OpenTelemetry.Enabled && e.ApiDefinition.DetailedTracing {
 		ctx, span := e.OpenTelemetry.TracerProvider.Tracer().Start(r.Context(), "GraphqlMiddleware Validation")
 		defer span.End()
+
+		operationType, err := gqlRequest.OperationType()
+		if err != nil {
+			e.logger.Debug("error while getting operation type for trace", abstractlogger.Error(err))
```

Comment:
It seems that we need to stop execution here by returning an error.

Reply:
thanks, that was an oversight. Corrected